index.js

```diff
@@ -21,7 +27,23 @@ module.exports.transform = ({ data, log, options }) => {
 
       return result.concat(writer);
     } catch (error) {
-      console.log(error);
+      const objectDetails =
+        object && object.__metadata && object.__metadata.id
+          ? ` (Object ID: ${object.__metadata.id})`
+          : "";
+
+      if (error.message === "ERROR_FAILED_SLUGIFY") {
+        log(
+          `Could not write object to disk because \`slugify()\` was used on an empty field.${objectDetails}`,
+          "fail"
+        );
+
+        debug(error);
+      } else {
+        log(`Could not write object to disk.${objectDetails} `, "fail");
```

Comment:
Will it be useful to append `error.message` to this log?

Reply:
I think I'll leave it out for now to avoid polluting the output too much. The full error message is available via the [debugging mechanism](https://github.com/stackbithq/sourcebit/wiki/Debugging).